bellman/src/gadgets/uint32.rs

```diff
@@ -69,14 +69,13 @@ impl UInt32 {
             })
             .collect::<Result<Vec<_>, SynthesisError>>()?;
 
-        Ok(UInt32 {
-            bits: bits,
-            value: value,
-        })
+        Ok(UInt32 { bits, value })
     }
 
-    pub fn into_bits_be(&self) -> Vec<Boolean> {
-        self.bits.iter().rev().cloned().collect()
+    pub fn into_bits_be(self) -> Vec<Boolean> {
```

Comment:
Is there a distinction between `to_bits_` and `into_bits_`? I am comparing this to the name change in `bellman/src/gadgets/num.rs`.

Reply:
The clippy warning that this change addresses is:
```
warning: methods called `into_*` usually take self by value; consider choosing a less ambiguous name
  --> bellman/src/gadgets/uint32.rs:78:25
   |
78 |     pub fn into_bits_be(&self) -> Vec<Boolean> {
   |                         ^^^^^
   |
   = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#wrong_self_convention
```

This follows the corresponding entry in the [Rust style guide](https://doc.rust-lang.org/1.0.0/style/style/naming/conversions.html).

I altered the various methods to be consistent with how they used `self` in context. I first tried having the `into_*` methods consume `self`, and where that was not possible due to the caller relying on passing an immutable reference, I altered the method name to `to_*`.